Formula/botan.rb

```diff
@@ -25,6 +25,7 @@ class Botan < Formula
   depends_on "pkg-config" => :build
   depends_on "python@3.11"
   depends_on "sqlite"
+  depends_on xcode: "13.3"
```

Comment:
Not sure this is right, since this will require a full Xcode install, when it seems like a new enough CLT suffices.

Reply:
upstream needs xcode 13.3, so I figured this dependency declaration would be the route, any thoughts?

Reply:
Does it not work with the CLT?

Reply:
It should. Let's just use `fails_with` instead of depending on a full Xcode installation.